index.html

```diff
@@ -10,49 +10,162 @@
+                      <li><button type="button">javaScript</button></li>
+                 </ul>
+                  </div>
+                  <div class="project-check"> <button type="button">See Project</button> </div>
+              </div>
+           </div>
+
+           <div class="card-3">
+            <img src="Images/Snapshoot-Portfolio-image-3.png" alt="" class="card__image">
+            <div class="class__content">
+           <h2 class="big-txt">Tonic</h2>
+           <div class="tag-group">
+            <p class="tag-main">CANOPY</p>
+            <div class="separator"></div>
+            <p class="tag">Back End Dev</p>
+            <div class="separator"></div>
+            <p class="tag">2015</p> </div>
+             <div class="card-txt"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+             <div class="prog">
+              <ul>
+                <li><button type="button">html</button></li> 
+                <li><button type="button">css</button></li>
+                <li><button type="button">javaScript</button></li>
+             </ul>
+              </div>
+              <div class="project-check"> <button type="button">See Project</button> </div>
+            </div>
+           </div>
+           <div class="card-4">
+           <img src="Images/Snapshoot-Portfolio-image-4.png" alt="" class="card__image">
+           <div class="class__content">
+           <h2 class="big-txt">Multi-Post Stories</h2>
+           <div class="tag-group">
+            <p class="tag-main">CANOPY</p>
+            <div class="separator"></div>
+            <p class="tag">Back End Dev</p>
+            <div class="separator"></div>
+            <p class="tag">2015</p> </div>
+            <div class="card-txt"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+            <div class="prog">
+              <ul>
+                <li><button type="button">html</button></li> 
+                <li><button type="button">css</button></li> 
+                <li><button type="button">javaScript</button></li>
+             </ul>
+              </div>
+              <div class="project-check"> <button type="button">See Project</button> </div>
+          </div>
+          </div>
        </section>
 
-      </div>
+          <section class="headline-2">
+            <div class="Greetings-2-txt">
+               <h2 class="cheers-2-txt">About<br>Myself</h2>
+            </div>
+            <div class="main-headline-txt">
+              <p>
+                I am a beginner student in the Microverse Program, this is my first week!
+                I'm learning right now how to implement flexbox properties in CSS. 
+                This is my first project using flexbox and setting up a mobile version skeleton
+              </p>
+              <div class="connect">
+                   <button class="small-txt" type="button">LET'S CONNECT</button>
```

Comment:
- [ ] The `LET'S CONNECT` is not a button is just a heading, please kindly use the `<hx>` tag.
- [ ] Also kindly note that according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). I have noticed that this title doesn't have the same color like the title in the Figma design.

Your design
![Screenshot from 2022-07-02 19-03-14](https://user-images.githubusercontent.com/74501523/177009705-48481902-2abc-4de4-bf9c-2f25d2e6857d.png)

Figma design
![Screenshot from 2022-07-02 19-07-18](https://user-images.githubusercontent.com/74501523/177009830-11c7f341-bc9d-42b5-b3e9-d5c7a6aa4d2c.png)